extensions-contrib/opentelemetry-emitter/README.md

```diff
@@ -0,0 +1,118 @@
+# OpenTracing Emitter
+
+The OpenTelemetry emitter generates OpenTelemetry Spans for queries.
+
+
+## Configuration
+
+### Enabling
+
+Load the plugin and enable the emitter in `common.runtime.properties`.
+
+Load the plugin:
+
+```
+druid.extensions.loadList=[..., "opentelemetry-emitter"]
+```
+
+Enable the emitter:
+
+```
+druid.emitter=opentelemetry
+```
+
+*_Don't forget to remove the line_ `druid.emitter=noop` _to avoid rewriting_.
```

Comment:
not sure what we mean by that "rewriting". Maybe we can link to the common emitter configuration docs for further reading?